script.lua

```diff
@@ -6349,37 +6337,87 @@ function setupColor(table, color)
         colorTint = Color.fromHex(Tints[color].Presence)
     end
     table.setColorTint(colorTint)
-    table.UI.setXml("")
```

Comment:
why are you removing this line?

Reply:
Because `setupSwapButtons` is going to set entirely new UI xml below.